requre/__init__.py

```diff
@@ -0,0 +1,3 @@
+from requre.constants import VERSION_REQURE_FILE
+
+_ = VERSION_REQURE_FILE
```

Comment:
?

Reply:
to avoid pre-commit issues with unused import. 
@lachmanfrantisek suggested to use it in same way how it is used eg in ``packit`` but it uses ``__all__ = [ class.__name__]`` syntax, but I have no idea if it is possible to get the name of basic classes (in this case ``int``) to do it in the same way.

Reply:
Is it neaded in the `__init__`? (= Will this be imported outside of the requre code-base?)

Reply:
not sure, just I saw similar way in rebaseHelper :-)